src/plugins/discover/public/kibana_services.ts

```diff
@@ -15,21 +15,24 @@ import { createGetterSetter } from '../../kibana_utils/public';
 import { search } from '../../data/public';
 import { DocViewsRegistry } from './application/doc_views/doc_views_registry';
 
-let angularModule: any = null;
+let angularModule: ng.IModule | null = null;
 let services: DiscoverServices | null = null;
 let uiActions: UiActionsStart;
 
 /**
  * set bootstrapped inner angular module
  */
-export function setAngularModule(module: any) {
+export function setAngularModule(module: ng.IModule) {
   angularModule = module;
 }
 
 /**
  * get boostrapped inner angular module
  */
-export function getAngularModule() {
+export function getAngularModule(): ng.IModule {
+  if (!angularModule) {
```

Comment:
:information_source: Adding the same runtime check that we have for `getServices` so that TS knows this is not nullable anymore.